### DISCUSSION TITLE
Does tailwind support PHP?

### DISCUSSION THREAD

**Discussion by lassejlv** [tailwindlabs/tailwindcss#7223] [Help]
Hello there **Tailwind Support Team**

Does tailwind support like index.php

Like this 

```bash 
content: ["./src/**/*.{html,js,php}", "./index.php"],
```

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

Yep, it supports any language! Are you experiencing any issues?

**Reply by lassejlv** [reply]
I fixed IT but thAnks 😃

**Reply by lassejlv** [reply]
@RobinMalfait have a quick question.

Can i also use it to express-ejs ?
